### PR TITLE
feat: pr, issues, discussions and releases webhook handling

### PR DESCRIPTION
**Pull request by danieldietzler**

### CODE REVIEW

File: src/services/webhook.service.ts [hunk 1]
```diff
@@ -142,4 +167,47 @@ export class WebhookService {
       flags: [MessageFlags.SuppressNotifications],
     });
   }
+
+  private getEmbed(
+    action: 'opened' | 'closed' | 'created' | 'deleted' | 'answered',
+    repositoryName: string,
+    title: string,
```
Comment (jrasm91) [inline src/services/webhook.service.ts:174]: these are essentially all strings, so you might consider doing something like `({ action: string, repositoryName: string, title: string, event: BaseEvent })` instead.

**Review by jrasm91 — APPROVED** [review]
Nice! Although I don't see any tests :cry:

File: src/services/webhook.service.ts (hunk 1 above)
Comment (danieldietzler) [reply src/services/webhook.service.ts:174]: Yeah I've been thinking about that, but then I'd also need a default case in the switch :P

**Comment by danieldietzler** [comment]
> Nice! Although I don't see any tests 😢

Oh right tests is something we could/should do now... Problem for a future Daniel :)

File: src/services/webhook.service.ts (hunk 1 above)
Comment (danieldietzler) [reply src/services/webhook.service.ts]: Other changes made this even more natural, so I changed it.

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**